task/06-conditions-n-loops-tasks.js

```diff
@@ -226,7 +258,7 @@ function reverseString(str) {
  *   34143 => 34143
  */
 function reverseInteger(num) {
-    throw new Error('Not implemented');
+    return Number(num.toString().split("").reverse().join(""));
```

Comment:
Don't use `Number` to convert string to number)) Use unary `+` or `parseFloat/parseInt`.